src/lib/input_schema.js

```diff
@@ -0,0 +1,78 @@
+const fs = require('fs');
+const path = require('path');
+const { ACTOR_SPECIFICATION_FOLDER } = require('./consts');
+const outputs = require('./outputs');
+const { getLocalConfig } = require('./utils');
+
+const DEFAULT_INPUT_SCHEMA_PATHS = [
+    '.actor/INPUT_SCHEMA.json',
+    './INPUT_SCHEMA.json',
+];
+
+const readInputSchemaOnPath = async (inputSchemaPath) => {
+    outputs.info(`Reading input schema stored at ${inputSchemaPath}`);
+    if (!fs.existsSync(inputSchemaPath)) {
+        throw new Error(`Input schema has not been found at ${inputSchemaPath}.`);
+    }
+    try {
+        const inputSchema = JSON.parse(fs.readFileSync(inputSchemaPath).toString());
```

Comment:
There is `getJsonFileContent` in `lib/utils.js` that takes path and returns object or undefined if path does not exists